scripts/bsd/clear_users_and_ssh_keys.sh

```diff
@@ -19,9 +19,24 @@ fi
 # these users will exist in postgres images.
 # we need to remove these users.
 
-project_level_ssh_keys=$(curl -fsH "Metadata-Flavor: Google"  http://metadata.google.internal/computeMetadata/v1/project/attributes/ssh-keys)
-instance_level_ssh_keys=$(curl -fsH "Metadata-Flavor: Google"  http://metadata.google.internal/computeMetadata/v1/instance/attributes/ssh-keys)
-ssh_keys=$(printf '%s\n%s' "$project_level_ssh_keys" "$instance_level_ssh_keys")
+ssh_keys=""
+
+# do not fail if curl fails with 404
+curl_helper ()
+{
+    exit_code=0
+    curl_output=$(curl -fsSH "Metadata-Flavor: Google" $1 2>&1) || exit_code=$?
```

Comment:
Why not just do this portion unconditionally? I.e.
```bash
    curl_output=$(curl -fsSH "Metadata-Flavor: Google" $1 2>&1)
    exit_code=$?
```

Reply:
Hm, actually, there might be a better option:
```bash
curl --fail --silent -o /tmp/test -w "%{http_code}\n" 'https://anarazel.de/nofile'
404
```
```bash
curl --fail --silent -o /tmp/test -w "%{http_code}\n" 'https://anarazel.de/'
200
```

Reply:
> Hm, actually, there might be a better option:
> 
> ```shell
> curl --fail --silent -o /tmp/test -w "%{http_code}\n" 'https://anarazel.de/nofile'
> 404
> ```
> 
> ```shell
> curl --fail --silent -o /tmp/test -w "%{http_code}\n" 'https://anarazel.de/'
> 200
> ```

That looks cleaner but then curl's output needs to be gotten from `/tmp/test` file and this file needs to be removed as well. Would you prefer that approach?

Reply:
Yes - I'd not use /tmp/test though. Use `mktemp` or such.